aten/src/ATen/native/cpu/UpSampleKernel.cpp

```diff
@@ -1270,38 +1268,17 @@ struct HelperInterpCubic : public HelperInterpBase {
   // taken from
   // https://github.com/python-pillow/Pillow/blob/6812205f18ca4ef54372e87e1a13ce4a859434df/
   // src/libImaging/Resample.c#L46-L62
-  template<typename scalar_t>
+  template<typename scalar_t, bool use_keys_cubic=true>
```

Comment:
This may be a nit so feel free to ignore, but I think it won't be immediately obvious what "keys" refers to here, especially considering it's a common noun.
In my attempt at this I named it `use_0_5_cst` or something like that - not sure whether that's strictly better.